### PR TITLE
README: cut 0.5 release and update compatibility matrix

### PR DESCRIPTION
**Pull request by s-urbaniak**
Signed-off-by: Sergiusz Urbaniak <sergiusz.urbaniak@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -254,4 +255,4 @@ While the community has not yet fully agreed on alert severities and their to be
 ## Note
 
 You can use the external tool call [prom-metrics-check](https://github.com/ContainerSolutions/prom-metrics-check) to validate the created dashboards. This tool allows you to check if the metrics installed and used in Grafana dashboards exist in the Prometheus instance. 
-Please have a look at https://github.com/ContainerSolutions/prom-metrics-check.
\ No newline at end of file
+Please have a look at https://github.com/ContainerSolutions/prom-metrics-check.
```
Comment (s-urbaniak) [inline README.md:258]: I blame emacs :man_shrugging:

File: README.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ A set of Grafana dashboards and Prometheus alerts for Kubernetes.
 | release-0.2  | v1.14.1 and before | v2.11.0+  |
 | release-0.3  | v1.17 and before   | v2.11.0+  |
 | release-0.4  | v1.18              | v2.11.0+  |
+| release-0.5  | v1.19              | v2.11.0+  |
```
Comment (brancz) [inline README.md:16]: if `master` previously was `v1.18+`, why is release-0.5 v1.19 now?

File: README.md (hunk 1 above)
Comment (s-urbaniak) [reply README.md:258]: (no net change)

File: README.md (hunk 2 above)
Comment (s-urbaniak) [reply README.md:16]: whoopsie, updated master to point to 1.19, great catch :+1:

Comment (lilic) [reply README.md:16]: I am not sure things changed that we need to not support 1.18? Were there any metric changes? Tests seem to still pass?

Comment (s-urbaniak) [reply README.md]: @lilic @brancz adapted to `v1.18+` only, PTAL, I guess that's what you meant.

File: README.md [hunk 3]
```diff
@@ -13,6 +13,7 @@ A set of Grafana dashboards and Prometheus alerts for Kubernetes.
 | release-0.2  | v1.14.1 and before | v2.11.0+  |
 | release-0.3  | v1.17 and before   | v2.11.0+  |
 | release-0.4  | v1.18              | v2.11.0+  |
+| release-0.5  | v1.18+             | v2.11.0+  |
```
Comment (povilasv) [inline README.md:16]:
```suggestion
| release-0.5  | v1.19+             | v2.11.0+  |
```

Comment (povilasv) [reply README.md:16]: Or I don't see the point in this release?

File: README.md (hunk 2 above)
Comment (povilasv) [reply README.md]: so why release a new version and not push changes to release-0.4?

Comment (lilic) [reply README.md]: We do this when a new version of k8s is cut, so now that 1.19 is cut. So its more like 1.18 and 1.19 and previous versions might not work for 1.19. Hope that makes sense?

File: README.md (hunk 3 above)
Comment (s-urbaniak) [reply README.md:16]: As discussed on Kubernetes slack we'll set `v1.19` and reserve `release-0.6` for `v1.19+`

Comment (s-urbaniak) [reply README.md:16]: xref https://kubernetes.slack.com/archives/CAX9GU941/p1596108610030400

**Pull request merged**